src/tests/news/search.test.ts

```diff
@@ -0,0 +1,96 @@
+import request from 'supertest';
+import app from '../../index';
+
+const getSearchResultsUrl = '/api/news/search';
+
+describe('GET /api/news/search', () => {
+  describe('Validation', () => {
+    it('should validate required q parameter and return a 400 if q is missing', async () => {
```

Comment:
i would consider an empty line before each `it` for readability, as you have in some places